app/decorators/conviction_decorator.rb

```diff
@@ -12,4 +12,13 @@ def custodial_sentence?
       ConvictionType::ADULT_CUSTODIAL_SENTENCE,
     ].include?(self)
   end
+
+  def motoring?
```

Comment:
Please use the inquirer as per example in my last PR. I specifically use motoring as the example.

No need to create any new method.
All motoring convictions are part of parent group, so inquiring the parent will give you what you need.